html/docs/include/features.php

```diff
@@ -907,14 +907,80 @@
-        find out when the PHP internal garbage collector triggers, how many variables
-        it was able to clean up, how long it took and how how much memory was actually freed.',
-        ""
-    ),
+	'garbage_collection' => array(
+		'Garbage Collection Statistics',
+		FUNC_GARBAGE_COLLECTION,
+		'Xdebug\'s built-in garbage collection statistics profiler allows you to
+		find out when the PHP internal garbage collector triggers, how many variables
+		it was able to clean up, how long it took and how how much memory was actually freed.',
+		'
+<h2 name="introduction">Introduction</h2>
+
+<p>Garbage Collection (GC) in PHP can have a serious impact on memory and performance,
+so understanding when it is triggered and how efficient each run is allows
+you to optimise your programs. The PHP Engine does not provide a mechanism to
+gather statistics about garbage collection, but xdebug now provides access
+to these statistics since version 2.6.
+</p>
+
+<p>For the time being, the garbage collection statistics are collected in a human
+readable, tabular format only, because there are no tools for this kind of report
+that we could generate machine readable output for. Future versions may include
+exports in other formats to allow machine processing.</p>
+
+<h2 name="usage">Usage</h2>
+
+<p>There are two approaches to start collecting the GC statistics with two different
+use-cases. The first is entirely via INI settings and the primary use-case is to
+collect statistics for indiviual CLI script runs (where GC is often an issue).</p>
+
+<p><code>php -dxdebug.gc_stats_enable=1 your_script.php</code></p>
+
+<p>If you wish to collect the garbage collection statistics for every script
+that you execute, you can set the [CFG:gc_stats_enable] INI setting on the system
+or directory-level. Be aware that activating the collection globally will produce
+an output file for *every* executed script, even if the garbage collector didn\'t
+run.</p>
+
+<p>The second approach to starting collection is to call the function
+<code>xdebug_start_gcstats</code> directly in your PHP script. This gives you
+more control over when collection is started.</p>
+
+<p>You can stop collection for both INI and function based approaches by
+calling <code>xdebug_stop_gcstats</code>.</p>
+
+<h2 name="output-tabular-text">Tabular Text (Human Readable)</h2>
+
+<p>The default (and only for now) output format of Garbage Collection statistics
+is a tabular human readable text output.</p>
+
+<p><code><pre>
+Garbage Collection Report
+version: 1
+creator: xdebug 2.6.0beta2-dev (PHP 7.2.0RC5)
+
+Collected | Efficiency% | Duration | Memory Before | Memory After | Reduction% | Function
+----------+-------------+----------+---------------+--------------+------------+---------
+    10000 |    100.00 % |  0.00 ms |       5539880 |       579880 |   -10.47 % | bar
+	10000 |    100.00 % |  0.00 ms |       5540040 |       580040 |   -10.47 % | Garbage::produce
```

Comment:
Is the indentation right between those two? As this is straight output, feel free to use spaces (for alignment).